### PR TITLE
[13.0][FIX] account_invoice_fiscal_position_update: tax included

### PR DESCRIPTION
**Pull request by jjscarafia**
When any involved tax has the option price_include we need to recompute price_subtotal.

### CODE REVIEW

**Comment by pablohmontenegro** [comment]
@rafaelbn hi Rafael, could you please review if this this pull request can be merged?

**Comment by pablohmontenegro** [comment]
@chienandalu @pedrobaeza hi, could you please review if this pull request can be merged?

**Comment by pedrobaeza** [comment]
Sorry, not using this module.

**Comment by pablohmontenegro** [comment]
ping @OCA/accounting-maintainers

**Comment by pablohmontenegro** [comment]
@alexis-via @dreispt hi, could you please review if this pull request can be merged?

File: account_invoice_fiscal_position_update/models/account_invoice.py [hunk 1]
```diff
@@ -47,7 +47,15 @@ def fiscal_position_change(self):
...
                     taxes = fp.map_tax(taxes)
 
+                # if we are replacing or adding a tax with "price_include" we
+                # should recompute price_subtotal
+                recompute_subtotal = any(
+                    x.price_include for x in (taxes + line.tax_ids)
+                )
+
                 line.tax_ids = [(6, 0, taxes.ids)]
+                if recompute_subtotal:
```
Comment (SkiBY) [inline account_invoice_fiscal_position_update/models/account_invoice.py:57]: But why we need to check if there are price_include taxes only?
if you will return to taxes out, there will be the same error
so you could remove recumpute subtotal check and live _onchange_price_subtotal call only

**Comment by jjscarafia** [comment]
closing as it is no happening anymore on future versions (at lease 16+)

**Pull request closed without merging**